src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs

```diff
@@ -179,7 +179,7 @@ impl<const NULLABLE_RESULT: bool> AggregateFunction for AggregateNullUnaryAdapto
                     return Ok(());
                 }
 
-                if unsafe { v.get_bit_unchecked(row) } {
+                if v.get_bit(row) {
```

Comment:
Why remove unsafe?

Reply:
Seems `get_bit` has same behavior as `get_bit_unchecked` without using unsafe.

Reply:
`get_bit_unchecked`'s performance is better

Reply:
fixed.